backend/services/inventory/api/http/api_inventory_test.go

```diff
@@ -14,27 +14,30 @@
 package http
 
 import (
+	"bytes"
 	"context"
 	"encoding/base64"
 	"encoding/json"
 	"fmt"
+	"io"
 	"net/http"
+	"net/http/httptest"
 	"reflect"
 	"strconv"
 	"strings"
 	"testing"
 	"time"
 
-	"github.com/ant0ine/go-json-rest/rest"
 	"github.com/ant0ine/go-json-rest/rest/test"
```

Comment:
The dependency is still here :point_up:
```suggestion
```
It's used here: https://github.com/mendersoftware/mender-server/blob/c8d7ebff5c9addee9c266a6e05f62fde339297f3/backend/services/inventory/api/http/api_inventory_test.go#L3138-L3142